pkg/auth/providers/ldap/ldap_provider.go

```diff
@@ -245,3 +278,11 @@ func (p *ldapProvider) getDNAndScopeFromPrincipalID(principalID string) (string,
 	externalID := strings.TrimPrefix(parts[1], "//")
 	return externalID, scope, nil
 }
+
+// if provider only enabled by another provider using ldap for group search e.g. SAML
+func (p *ldapProvider) groupSearchProvider() bool {
```

Comment:
now that we use ldap for searching users too, can we rename `groupSearchProvider` to something that says that ldap is being used for saml search? maybe `calledForSamlSearch` ?

Reply:
samlSearchProvider